### PR TITLE
Add "gpt-3.5-turbo-16k" model chat_completion.rs

### PR DESCRIPTION
**Pull request by dejavu1987**
I needed this 16k context model but saw it is not added.

### CODE REVIEW

File: src/v1/chat_completion.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ use crate::impl_builder_methods;
 use crate::v1::common;
 
 pub const GPT3_5_TURBO: &str = "gpt-3.5-turbo";
+pub const GPT3_5_TURBO_16K: &str = ""gpt-3.5-turbo-16k"";
```
Comment (dongri) [inline src/v1/chat_completion.rs:9]: `""gpt-3.5-turbo-16k""` -> `"gpt-3.5-turbo-16k"` ?

Comment (dejavu1987) [reply src/v1/chat_completion.rs:9]: Oh sorry for that :)

Comment (dejavu1987) [reply src/v1/chat_completion.rs]: Fixed, thanks for your quick response.

**Review by dongri — APPROVED** [review]

**Comment by dongri** [comment]
[LGTM](https://lgtm.lol/i/807)

**Pull request merged**